pkg/term/term.go

```diff
@@ -26,10 +29,12 @@ type Winsize struct {
 	y      uint16
 }
 
+// StdStreams returns the standard streams (stdin, stdout, stedrr)
 func StdStreams() (stdIn io.ReadCloser, stdOut, stdErr io.Writer) {
 	return os.Stdin, os.Stdout, os.Stderr
 }
 
+// GetFdInfo returns the file description information of the given input (eg a file)
```

Comment:
"GetFdInfo returns the file descriptor for an os.File, and indicates whether the file represents a terminal"